main.py

```diff
@@ -13,17 +14,18 @@ def sign_up() -> Optional[USER]:
         login = input("Enter Login(q to quit): ")
         if login == "q":
             return None
+        user_id = User_Menager().get_max_user_id() + 1
```

Comment:
here i just get max id from DB and create new one which is max +1

Reply:
u don't need this if you setup auto incremental